x/batch/decorator_test.go

```diff
@@ -69,14 +95,15 @@ func mockTags(num int64) []common.KVPair {
 	return list
 }
 
-func mockData(num int64, content []byte) *batch.ByteArrayList {
+func mockData(num int64, content []byte) []byte {
```

Comment:
Name could be improved. `mockData` is not helpful to someone reading this code as it does not explain what is returned.

Reply:
It actually mocks Data, it makes total sense when you see where it is used.

I'd also like to keep the scope of this refactoring to a minimum - basically refactoring out testify and convey.